generate_wordstr_box.py

```diff
@@ -44,6 +38,9 @@ def trim(im):
 # create WordStr line boxes for Indic & RTL
 for line in lines:
     line = unicodedata.normalize('NFC', line.strip())
+    if args.rtl:
+        line = line.translate(str.maketrans("()[]{}»«><", ")(][}{«»<>"))
+        line = bidi.algorithm.get_display(line)
     if line:
         print("WordStr 0 0 %d %d 0 #%s" % (width, height, line))
-        print("\t 0 0 %d %d 0" % (width, height))
+        print("\t 0 0 %d %d 0" % (width, height), end='')
```

Comment:
This suppresses the LF, so the file size is reduced by one byte. Does it also have an effect on the training?

Reply:
I was getting some unexpected results and hence had added that. It might have been when I was using files generated by text2image.

Reply:
Now I remember, it was needed for the ground-truth text files. 
Not required for box files.

Reply:
So ground truth text files must not end with a line feed? What happens if they do? Some editors add a line feed each time when a file was changed and saved.